src/components/vanilla/charts/DownloadButton/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { DataResponse, DimensionOrMeasure } from '@embeddable.com/core';
+import downloadAsCSV from '../../../util/downloadAsCSV';
+import Container from '../../Container';
+import Button from '../../Button';
+
+import { REGULAR_FONT_SIZE } from '../../../constants';
+import Spinner from '../../Spinner';
+
+
+export type Props = {
+  columns: DimensionOrMeasure[];
+  results: DataResponse;
+  buttonLabel: string;
+};
+
+
+
```

Comment:
remove this empty lines.

do you have prettier installed?  this should format automatically when saving

Reply:
@mad-raccoon. @hjkmarshall I think we can add `prettier` to "test" CI/CD step and catch these errors before they reach code review?

wdyt?

Reply:
Yes. We have the same on the FE repo